### ISSUE TITLE
preprocess of input frame is different from original implementation.

### ISSUE THREAD

**Issue by hzhang57** [hassony2/kinetics_i3d_pytorch#11]
Hi, @hassony2 
Thank you for sharing the pytorch-i3d implementation.
I find the preprocess of input frame is different from the original implementaion.
In I3D-tensorflow version, they state  that "For RGB, ..... . Pixel values are then rescaled between -1 and 1. "
However, in i3d_pt_profiling.py,  I only see process like
`    dataset = datasets.ImageFolder(dataset_path,
                                   transforms.Compose([
                                       transforms.CenterCrop(args.im_size),
                                       transforms.ToTensor(),
                                       normalize,
                                   ]))`

But in the other implementations (https://github.com/piergiaj/pytorch-i3d), they use 
`img = (img/255.)*2 - 1`

I am wondering whether this inconsistent will lead to a drop of performance.
Thanks

**Comment by hassony2** [comment]
Hi @hzhang57,

The profiling code from i3d_pt_profiling was just meant for profiling purposes :). Therefore I feed in dummy data and don't worry about the output values.
If you want the proper demo you should look at i3d_pt_demo where you can verify that indeed the input is scaled between [-1, 1].
For proper results you should indeed rescale the inputs.

Hope this helps!

All the best,

Yana

**Comment by xb-chang** [comment]
Hi, @hassony2 , 
Thank you so much for your post.

I ran the pt_demo code.
I try to parse the rgb.npy file in data/ provided by you. I assume that you are preprocessing each frame with 
img = (img/255.)*2 - 1
right?
However, I can not recover the images from such rgb.npy.
But I can recover the images in flow.npy.

Could you please tell me how to get the rgb input data for your code?

Regards,
Xiaobin
